openslides_backend/models/motion_category.py

```diff
@@ -0,0 +1,40 @@
+from ..shared.patterns import Collection
+from . import fields
+from .base import Model
+
+
+class MotionCategory(Model):
+    """
+    Model for motion category.
+
+    Reverse fields:
+    - child_ids: (motion_category/parent_id)[]
+    - motion_ids: (motion/category_id)[]
+    """
+
+    collection = Collection("motion_category")
+    verbose_name = "motion_category"
+
+    id = fields.IdField(description="The id of this motion category.")
+    meeting_id = fields.RequiredForeignKeyField(
+        description="The id of the meeting of this motion_category.",
+        to=Collection("meeting"),
+        related_name="category_ids",
+    )
+    name = fields.RequiredCharField(
+        description="The name of this motion category.", maxLength=255
+    )
+    prefix = fields.RequiredCharField(
+        description="The prefix of this motion category.", maxLength=32
+    )
+    weight = fields.PositiveIntegerField(
+        description="The weight of this motion category"
+    )
+    level = fields.PositiveIntegerField(
```

Comment:
level is a calucated field I think. Please remove it here and add a TODO that these calculated fields have to be coded later. Please also add a Pull Request to the main repo to fix the models.txt and add the `// calulated` comment

Reply:
Done.